schemas/coverage/coverage/coverage.1.schema.json

```diff
@@ -16,8 +16,11 @@
       "type": "string"
     },
     "telemetryEnabled": {
-      "type": "integer",
-      "enum": [0, 1]
+      "enum": [
```

Comment:
Did our formatting change? I can  remove this if needed.

Reply:
Either the pretty-printing changed a bit or just didn't get checked in last time. There's [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1383111) on file to enforce consistency here.

Let's keep this fix, thanks!